forest/bonsai/tests/test_simulate_gps_data.py

```diff
@@ -0,0 +1,60 @@
+"""
+Tests for simulate_gps_data
+module
+"""
+
+import pytest
+
+from forest.bonsai.simulate_gps_data import get_path
+
+
+@pytest.fixture(scope="session")
+def path1():
+    """Random path"""
+
+    lat1 = 51.458498
+    lon1 = -2.59638
+    lat2 = 51.457619
+    lon2 = -2.608466
+    path, dist = get_path(
+        lat1,
+        lon1,
+        lat2,
+        lon2,
+        "car",
+        "5b3ce3597851110001cf6248551c505f7c61488a887356ff5ea924d5",
+    )
+
+    return path, dist
+
+def test_get_path_simple_case1(path1):
+    """Tests ending lattitude."""
+
+    assert path1[0][0][1] == 51.458498
+
+def test_get_path_simple_case2(path1):
```

Comment:
If test function names are self-describing then you won't need a doctring and it's easier to see what was the failure in a test run. For example: `test_get_path_ending_longitude`